### DISCUSSION TITLE
How to update from 2.0 version to 2.6 on linux?

### DISCUSSION THREAD

**Discussion by theteju** [alessandrofrancesconi/gimp-plugin-bimp#322] [Q&A] [answered]
I successfully install Bimp on gimp 2.10. But I accidentally install 2.0 version. How can I update the plug in to 2.6 (latest) version?

**Comment by alessandrofrancesconi** [comment]
How did you install it and on which distro? 
I assume you are not installing it via source code as described in Readme (with this method is guaranteed you will have the latest version)

**Comment by theteju** [comment]
Thank you for your response. I am on kde neon, debian based.

I followed instructions. Make and sudo make install-admin that's how I
installed.

But I didn't realize version 2.6 was there. I downloaded 2.0 zip source
code and installed it.

Teju.

**Comment by alessandrofrancesconi** [comment, marked as answer]
Ok! So you just have to download the latest source code and do the procedure again.
https://github.com/alessandrofrancesconi/gimp-plugin-bimp/releases/latest

The older version with be automatically replaced.
